easybuild/easyconfigs/t/Trimmomatic/Trimmomatic-0.39-Java-17.eb

```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'Tarball'
+
+name = 'Trimmomatic'
+version = '0.39'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'http://www.usadellab.org/cms/?page=trimmomatic'
+description = """Trimmomatic performs a variety of useful trimming tasks for illumina 
+ paired-end and single ended data.The selection of trimming steps and their associated 
+ parameters are supplied on the command line. """
+
+toolchain = SYSTEM
+
+source_urls = ['http://www.usadellab.org/cms/uploads/supplementary/Trimmomatic/']
+sources = [SOURCE_ZIP]
+checksums = ['2f97e3a237378d55c221abfc38e4b11ea232c8a41d511b8b4871f00c0476abca']
+
+dependencies = [('Java', '17')]
```

Comment:
Maybe it's better to stick to `Java/11` here, so this easyconfig can be used as a dependency in an easyconfig using the `foss/2021b` toolchain (since we're already using `Java/11` in `R-4.1.2-foss-2021b.eb`)

Reply:
Oh, and we already have an easyconfig for `Trimmomatic` 0.39 with `Java/11`...

Can we make sure that `Trimmomatic` 0.39 actually works with `Java` 17.x?